src/ui/ui_pubkey_nbgl.c

```diff
@@ -70,12 +70,26 @@ static void confirmationChoiceClbk(bool confirm) {
     nbgl_useCaseStatus(confirm_text, confirm, ui_idle);
 }
 
+static void address_verification_cancelled(void) {
+    io_seproxyhal_touch_display_cancel(NULL);
+    nbgl_useCaseStatus("Address rejected", false, ui_idle);
+}
+
+static void ui_display_addr(void) {
+    nbgl_useCaseAddressConfirmation((char*) G_io_apdu_buffer + 200, &confirmationChoiceClbk);
+}
+
 static void warningChoiceClbk(bool reject) {
     if (reject) {
         io_seproxyhal_touch_display_cancel(NULL);
         nbgl_useCaseStatus("Address rejected", false, ui_idle);
```

Comment:
as per design guideline, you should replace `Address rejected` by `Address verification\ncancelled`